docs/navigation-configuration.md

```diff
@@ -1,242 +1,264 @@
 
-The following example shows the structure of different navigation paths. If the URL of the main application is `https://luigiexample.com`, then:
+Nodes belonging to the same view group are always rendered in their own view group iframe. Nodes not belonging to any view group follow the same-origin iframe rendering policy. 
 
-- `https://luigiexample.com/home/projects/pr1` reflects the `home/projects/pr1` navigation path. It is a valid navigation path as it exists in the defined navigation tree structure. The micro frontend view URL rendered in the content area is `https://my.microfrontend.com/projects/details.html#id=pr1`.
+The view groups feature also offers out-of-the-box caching. Each time you navigate to another view group, either a new iframe is created or it is reused if already exists. In both cases, the iframe you are navigating from becomes hidden and is available for you to use again. If you navigate back to the first iframe and it should be updated with new data, such when a new entry was added in the second iframe and you want to display it in a table in the first iframe, you must define a **preloadUrl** parameter for the view group under navigation.viewGroupSettings.
 
-- `https://luigiexample.com/home/maskopatol` defines an invalid `home/maskopatol` navigation path.
+You can also preload view groups. You just need to define which URL you want to preload, and Luigi will preload the view after some user interactions when the browser is most likely to be idle. This option is active by default, but you can deactivate it with a [configuration flag](navigation-parameters-reference.md#node-parameters).
 
-- `https://luigiexample.com/projects/pr1` defines the `projects/pr1` navigation path. It is not a valid navigation path, since a valid navigation path always starts from the root.
+For more information on setting caching with view refreshing and preloading for view groups, read [this document](navigation-parameters-reference.md#node-parameters).
 
+## Creating a dynamic path
+In Luigi, you can make a navigation path dynamically changeable according to your needs. This is accomplished by defining variables within the **pathSegement** or **viewUrl** navigation paths. 
 
-## Path parameters
+### pathSegment variables
+Instead of a static value for your **pathSegment**, you can add a colon to this value to make it act as a variable, for example `:userId`. This tells Luigi to accept any value for this **pathSegment**. 
```

Comment:
```suggestion
Instead of a static value for your **pathSegment**, you can add a colon to this value to make it act as a variable. For example, you can use `:userId`. This tells Luigi to accept any value for this **pathSegment**. 
```
We try to avoid `for example` in the middle of the sentence. We rather start with it and use `such as` in the middle. (https://kyma-project.io/community/guidelines/content#style-and-terminology-style-and-terminology-terminology)